File: 1/fizzbuzz.f

      program fizzbuzz

          integer i, finish
          i = 1
          finish = 100

          do while (i .LE. finish)
              if (MOD(i, 3) .EQ. 0) then
                  write(*, "(A)", advance="no") "Fizz"
              endif
              if (MOD(i, 5) .EQ. 0) then
                  write(*, "(A)", advance="no") "Buzz"
              endif
              if (MOD(i, 3) .NE. 0 .AND. MOD(i, 5) .NE. 0) then
                  write(*,"(I0)", advance="no") i
              endif
              write(*,"(A)") ""
              i = i + 1
          enddo

          stop
          end

File: 8/fizzbuzz.pl
#/usr/bin/env prolog

fb_cond(N) :- 0 is mod(N, 15) -> write('fizzbuzz\n').
fb_cond(N) :- 0 is mod(N, 3) -> write('fizz\n').
fb_cond(N) :- 0 is mod(N, 5) -> write('buzz\n').
fb_cond(N) :- write(N), write('\n').

fizzbuzz(0).
fizzbuzz(N) :- N1 is N - 1, fizzbuzz(N1), fb_cond(N).

run :- fizzbuzz(1000).
